src/main/java/aquarium/shop/order/OrderUser.java

```diff
@@ -7,7 +7,7 @@
 public class OrderUser {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
If we use SEQUENCE strategy for other entities maybe do same for all entities in this hierarchy.

Reply:
Not needed - discussed offline.